example_workspace/tutorial.hlasm

```diff
@@ -0,0 +1,95 @@
+* This is an example HLASM workspace
+
+* STATEMENTS
+* Source code is divided into statements
+* Each statement constists of:
+* - label/name (optional, yellow)
```

Comment:
label is not always optional. I would get rid of optional and mandatory field info at all.